test/Microsoft.ApplicationInsights.AspNetCore.Tests/project.json

```diff
@@ -0,0 +1,41 @@
+{
+  "version": "1.0.0-rc2",
+  "compilationOptions": {
+    "keyFile": "../../keys/35MSSharedLib1024.snk",
+    "delaySign": true,
+    "preserveCompilationContext": true
+  },
+  "dependencies": {
+    "Microsoft.ApplicationInsights.AspNetCore": "1.0.0-rc2-*",
+    "Microsoft.AspNetCore.Http.Features": "1.0.0-rc2-*",
+    "Microsoft.AspNetCore.Http": "1.0.0-rc2-*",
+    "Microsoft.AspNetCore.Http.Extensions": "1.0.0-rc2-*",
+    "Microsoft.AspNetCore.Hosting": "1.0.0-rc2-*",
+    "Microsoft.AspNetCore.Mvc.Core": "1.0.0-rc2-*",
+    "Microsoft.Extensions.Logging": "1.0.0-rc2-*",
+    "Microsoft.Extensions.Configuration.Json": "1.0.0-rc2-*",
+    "xunit": "2.1.0",
+    "FunctionalTestUtils": "1.0.0-rc2"
+  },
+  "testRunner": "xunit",
+  "commands": {
+    "test": "xunit.runner.aspnet"
+  },
+  "frameworks": {
+    "netstandardapp1.5": {
+      "imports": [
+        "dnxcore50",
+        "dotnet5.4",
+        "portable-net451+win8"
+      ],
+      "dependencies": {
+        "moq.netcore": "4.4.0-beta8",
+        "NETStandard.Library": "1.0.0-*",
```

Comment:
Removed net451 as I had some trouble running tests for that framework. We will add it back when we have a stable CLI version, and that is the reason why we have the dependencies left behind under netstandardapp!